okteto-pipeline.yaml

```diff
@@ -0,0 +1,7 @@
+icon: https://hub.litmuschaos.io/icons/litmus-header.svg
```

Comment:
Can this file be inside okteto directory

Reply:
No it has to be in root of any repo.. @rajdas98